packages/core/package.json

```diff
@@ -19,9 +19,34 @@
 	],
 	"type": "module",
 	"exports": {
-		".": "./lib/index.js",
-		"./ts-patch/install-patch": "./lib/ts-patch/install-patch.js",
-		"./ts-patch/install-patch-hooks": "./lib/ts-patch/install-patch-hooks.js"
+		".": {
+			"@flint.fyi/source": {
+				"types": "./.tsbuildinfo/types/src/index.d.ts",
+				"default": "./src/index.ts"
+			},
+			"default": "./lib/index.js"
+		},
+		"./ts-patch/install-patch": {
+			"@flint.fyi/source": {
+				"types": "./.tsbuildinfo/types/src/ts-patch/install-patch.d.ts",
+				"default": "./src/ts-patch/install-patch.ts"
+			},
+			"default": "./lib/ts-patch/install-patch.js"
+		},
+		"./ts-patch/install-patch-hooks": {
+			"@flint.fyi/source": {
+				"types": "./.tsbuildinfo/types/src/ts-patch/install-patch-hooks.d.ts",
+				"default": "./src/ts-patch/install-patch-hooks.ts"
+			},
+			"default": "./lib/ts-patch/install-patch-hooks.js"
+		},
+		"./ts-patch/proxy-program": {
+			"@flint.fyi/source": {
+				"types": "./.tsbuildinfo/types/src/ts-patch/proxy-program.d.ts",
+				"default": "./src/ts-patch/proxy-program.ts"
+			},
+			"default": "./lib/ts-patch/proxy-program.js"
+		}
```

Comment:
Needing this export when this file is an internal detail is the rational behind filing #1156.

Reply:
Actually refactored this to actually use the export, which is much nicer IMO.
Nonetheless, it does feel out place having these here, but I'm much less opposed to letting it stay in core if we have a good excuse for the bundler hacks (as we now do as of #1174).

Reply:
Fixed when merging in #1156.